Assets/Python/RiseAndFall.py

```diff
@@ -538,23 +538,23 @@ def adjust1700ADCulture(self):
 				if plot.getOwner() != -1:
 					utils.convertPlotCulture(plot, plot.getOwner(), 100, True)
 					
-		for x, y in [(48, 45), (50, 44), (50, 43), (50, 42), (49, 40)]:

```

Comment:
Does this work from the scenario file? I.e. is there no other culture left on those tiles so they won't flip shortly after?